plugin/src/main/groovy/com/bmuschko/gradle/tomcat/TomcatBasePlugin.groovy

```diff
@@ -57,7 +58,10 @@ class TomcatBasePlugin implements Plugin<Project> {
                 File webAppDir = getWarConvention(project).webAppDir
                 webAppDir.exists() ? webAppDir : null
             }
-            conventionMapping.map('classesDirectory') { project.sourceSets.main.output.classesDir.exists() ? project.sourceSets.main.output.classesDir : null }
+            conventionMapping.map('classesDirectory') {
```

Comment:
For this change we will have to consider all classes directories. Just picking the first one won't do. Could you please change the field in `TomcatRun` to use a `FileCollection` instead? Any handling of the field in the task needs to change accordingly.

Reply:
ok